modules/packages/ZMQ.chpl

```diff
@@ -165,13 +165,52 @@ versions may expose more features native to ZeroMQ.
 Serialization
 +++++++++++++
 
-In Chapel, sending or receiving messages on a :record:`Socket` uses
-`multipart messages <http://zguide.zeromq.org/page:all#Multipart-Messages>`_
-and the :chpl:mod:`Reflection` module to serialize primitive and user-defined
-data types whenever possible.  Currently, the ZMQ module serializes primitive
-numeric types, strings, and records composed of these types.  Strings are
-encoded as a length (as `int`) followed by the character array
-(in bytes).
+In Chapel, sending or receiving messages is supported for a variety of types.
+Primitive numeric types and strings are supported as the foundation.
+In addition, user-defined :type:`record` types may be serialized automatically
+as `multipart messages <http://zguide.zeromq.org/page:all#Multipart-Messages>`_
+by internal use of the :chpl:mod:`Reflection` module.
+Currently, the ZMQ module can serialize records of primitive numeric types,
+strings, and other serializable records.
+
+.. note::
```

Comment:
> Chapel 1.14/15 programs using ZMQ can't talk to Chapel 1.16 programs, though they can talk intra-version just fine.

This point made it clearer to me. I'm fine with referencing the version numbers for a caveat like this. I wonder if this text should be included in the first paragraph of the `.. note:`, e.g.

```rst

   The serialization protocol for strings changed in Chapel 1.16 in order to
   support inter-language messaging through ZeroMQ. (See
   :ref:`notes on interoperability <interop>` below.) As a result, 
   programs using ZMQ compiled with Chapel 1.15 or earlier can't 
   communicate with programs compiled with Chapel 1.16 or later, 
   though they can communicate intra-version just fine.

```